README.md

```diff
@@ -23,8 +23,19 @@ For the legacy Ladybug Grasshopper plugin see [this repository](https://github.c
 
 ## Usage
 
-```python
-# to be added
+```
+"""Get the percentage of time outdoor conditions are comfortable with/without sun + wind"""
+from ladybug.epw import EPW
+
+relative_path = './tests/epw/chicago.epw'
```

Comment:
This `relative_path` can get starters confused. Why not just use `weather_file` or something like that?